source/guides/stacks/mers.rst

```diff
@@ -208,45 +249,93 @@ This example shows how to perform transcoding with :command:`FFmpeg`.
 
       cp </path/to/video> ~/ffmpeg/input
 
-#. Run the *clearlinux/stack-mers* docker image, allowing shared access to the
-   workspace on the host:
-
+#. Run the *sysstacks/mers-clearlinux* docker image, allowing shared access to
+   the workspace on the host:
 
    .. code:: bash
 
       docker run -it \
-      -v ~/ffmpeg:/home/mers-user:ro \
-      clearlinux/stacks-mers:latest
+      --volume ~/ffmpeg:/home/mers-user:ro \
+      --device=/dev/dri \
+      --env QSV_DEVICE=/dev/dri/renderD128 \
+      sysstacks/mers-clearlinux:latest
+
+   .. note::
+
+      The :command:`--device` parameter and the **GSV_DEVICE** environment
+      variable allow shared access to the GPU on the host system. The values
+      needed may be different depending on host's graphics configuration.      
 
    After running the :command:`docker run` command, you enter a bash shell
    inside the container. 
 
-#. From the container shell, you can run :command:`FFmpeg` against the videos
-   in :file:`/home/mers-user/input` as you would normally outside of |MERS|.
+#. From the container shell, you can run :command:`FFmpeg` and
+   :command:`GStreamer` commands against the videos in
+   :file:`/home/mers-user/input` as you would normally outside of |MERS|.
 
-   For example, to transcode raw yuv420 content to SVT-HEVC and mp4:
+   Some sample commands are provided for reference. 
 
-   .. code:: bash
+   For more information on using the *FFmpeg* commands, refer to the `FFmpeg
```

Comment:
Line 278 and 281 - use the command role eg :command:`ffmpeg` and :command:`gstreamer`